<launch>



  <node pkg="multirobot_map_merge" type="map_merge" respawn="false" name="map_merge" output="screen">
    <param name="robot_map_topic" value="map"/>
    <param name="robot_namespace" value="robot"/>
    <param name="merged_map_topic" value="merged_map"/>
    <param name="world_frame" value="merged_map"/>
    <param name="known_init_poses" value="false"/>
    <param name="merging_rate" value="0.2"/>
    <param name="discovery_rate" value="0.05"/>
    <param name="estimation_rate" value="0.4"/>
    <param name="estimation_confidence" value="1.0"/>
  </node>

</launch>